jailbreaking/post-jailbreak/koreader.md

```diff
@@ -40,7 +40,7 @@ Your Kindle must be jailbroken and have MRPI and KUAL installed to be able to ru
                     <br/>
                     <code>kindle</code>: K4, K5, PW1
                     <br/>
-                    <code>kindlepw2</code>: All newer models (PW2, KV, KT2, PW3, KOA, KT3, KOA2, PW4, KT4, KOA3, PW5, KS) running firmware =< 5.16.2.1.1
+                    <code>kindlepw2</code>: PW2 and newer models running firmware =< 5.16.2.1.1
```

Comment:
imo this is less clear, but reasonable in the context. We tell people to take not of their model so the exhaustive list is useful, if difficult to keep updated